web/client/components/playback/Settings.jsx

```diff
@@ -121,73 +125,108 @@ export default ({
+                            <Message msgId="playback.settings.step.minute" msgParams={{ number: timeStep || 1 }}>{msg => <option value="minutes">{msg}</option>}</Message>
+                            <Message msgId="playback.settings.step.second" msgParams={{ number: timeStep || 1 }}>{msg => <option value="seconds">{msg}</option>}</Message>
+                        </FormControl>
+                    </Form>
+                </FormGroup>
+                <SwitchPanel onSwitch={(enabled) => toggleAnimationRange(enabled)} expanded={playbackRange.startPlaybackTime && playbackRange.endPlaybackTime} title={<Message msgId="playback.settings.range.title" />} buttons={playbackButtons}>
+                    <FormGroup controlId="formPlaybackMode" style={{margin: 10}}>
+                        <InlineDateTimeSelector
+                            tooltipId="playback.settings.range.animationStart"
+                            glyph="play"
+                            date={playbackRange.startPlaybackTime}
+                            onUpdate={startPlaybackTime => setPlaybackRange(getPlaybackRange({ ...playbackRange, startPlaybackTime }))}
+                            style={dateSelectorStyle}
+                            showButtons />
+                        <InlineDateTimeSelector
+                            glyph="stop"
+                            tooltipId="playback.settings.range.animationEnd"
+                            date={playbackRange.endPlaybackTime}
+                            onUpdate={endPlaybackTime => setPlaybackRange(getPlaybackRange({ ...playbackRange, endPlaybackTime }))}
+                            style={dateSelectorStyle}
+                            showButtons />
+                    </FormGroup>
+                </SwitchPanel>
+                <FormGroup controlId="formPlaybackFollowingMode">
+                    <Form componentClass="fieldset" inline>
+                        <ControlLabel><Message msgId="playback.settings.mode.following" />&nbsp;<InfoPopover text={<Message msgId="playback.settings.mode.followingDescription" />} /></ControlLabel>
+                        <span><SwitchButton checked={following} onChange={v => onSettingChange("following", v)}/></span>
+                    </Form>
+                </FormGroup>
+            </>
+        </Tab>
+        <Tab eventKey={2} title={<Message msgId="playback.settings.tab.layers" />}
+            style={{
+                height: 250,
+                width: 280,
+                overflowY: "auto",
+                paddingTop: 8
+            }}>
+            {layers.map(({title, id, checked} = {}) => {
+                const checkedLayer = layers?.filter(l=> l.checked);
+                const disable = (checkedLayer.length === 1 && checkedLayer?.[0]?.title === title) || selectedLayer === id;
+                return (
+                    <div className={"layer-setting"}>
+                        <label key={id}>{title}</label>
```

Comment:
- [ ] If the title layer is localized (from TOC), the application crashes. Please properly localize the title.

Reply:
Localized now